### PR TITLE
Update mentions to include names of people being mentioned/tagged

### PR DESCRIPTION
**Pull request by RtzLi**
Changed mentions from an array of ID's to an object containing the ID of the people tagged and the names that they were tagged with.

### CODE REVIEW

**Comment by Schmavery** [comment]
Can we help to clean this up a little?
You seem to do all the work to get the value for `JSON.parse(m.delta.data.prng)` 3 times. Is there some reason we can't just store that in a variable and then access several things from it? I feel like it would go a long way towards making this readable.

**Comment by RtzLi** [comment]
I've updated the code to look cleaner.

**Comment by Schmavery** [comment]
@cthuluhoop123 see any issues with this?
If not I will merge :)

**Comment by cthuluhoop123** [comment]
Seems fine to me :)

File: utils.js [hunk 1]
```diff
@@ -367,14 +367,22 @@ function formatAttachment(attachments, attachmentIds, attachmentMap, shareMap) {
 
 function formatDeltaMessage(m){
   var md = m.delta.messageMetadata;
+
+  var mdata = (m.delta.data === undefined) ? [] : (m.delta.data.prng === undefined) ? [] : JSON.parse(m.delta.data.prng);
+  var m_id = mdata.map(u => u.i);
+  var m_offset = mdata.map(u => u.o);
+  var m_length = mdata.map(u => u.l);
+  var mentions = {};
+  for (var i = 0; i < m_id.length; i++) mentions[m_id[i]] = m.delta.body.substring(m_offset[i], m_offset[i] + m_length[i]);
```
Comment (Schmavery) [inline utils.js]: Nitpick: Can you split this over 2 lines and surrounding `{` `}`? Thanks!

**Comment by Schmavery** [comment]
Thanks @RtzLi 😄

**Pull request merged**